test-snapshot-binaries/test-rpath-linux.py

```diff
@@ -1,25 +1,31 @@
 # Tests that DT_RPATH is correct for the dummy repl executable on Linux.
 # REQUIRES: platform=Linux
 # RUN: %{readelf} -d %{repl_swift} | %{FileCheck} %s
-# CHECK: {{.*}} {{\(RPATH\)|\(RUNPATH\)}} {{.*}}$ORIGIN/../lib/swift/linux
+# CHECK: {{.*}} {{\(RPATH\)|\(RUNPATH\)}} {{.*}}$ORIGIN/../lib/swift/linux/
```

Comment:
I wanted to use the `%linux_arch` substitution here too, but I don't see a way to do that for `CHECK` lines.